src/@types/users.ts

```diff
@@ -0,0 +1,23 @@
+export type Collaborator = SelectorItem<UserMini | GroupMini>;
+
+export type SelectorItem<T> = {
+    id: string;
+    item?: T; // ie UserMini or GroupMini
+    name: string;
+    value?: unknown;
+};
+
+export type UserMini = {
```

Comment:
I wonder if we should instead port these types from flow to typescript in BUIE and then import them here instead of duplicating the types across the two libraries?